MCV.js

```diff
@@ -0,0 +1,206 @@
+{
+	"translatorID": "b51ac026-ed35-4c68-89bb-b42b1e1ce8f2",
+	"label": "MCV",
+	"creator": "czar",
+	"target": "^https?://(www\\.)?mcvuk\\.com",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2018-07-05 01:11:13"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2018 czar
+	http://en.wikipedia.org/wiki/User_talk:Czar
+	
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+
+// attr()/text() v2 per https://github.com/zotero/translators/issues/1277
+function attr(docOrElem,selector,attr,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.getAttribute(attr):null;}function text(docOrElem,selector,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.textContent:null;}
+
+
+function detectWeb(doc, url) {
+	if (/\/(business|development|esports|influence)\//.test(url)) {
+		return "blogPost";
+	} else if (url.indexOf("/search?query=") != -1) {
+		return "multiple";
+	} else return null;
+}
+
+
+function scrape(doc, url) {
+	var translator = Zotero.loadTranslator('web');
+	translator.setTranslator('951c027d-74ac-47d4-a107-9c3069ab7b48'); // embedded metadata
+	translator.setDocument(doc);
+	
+	translator.setHandler('itemDone', function (obj, item) {
+		item.itemType = "blogPost";
+		item.publicationTitle = "MCV";
+		item.date = attr(doc,'meta[name="published"]','content');
+		item.title = item.title.replace(/- MCV$/,'');
+		if (item.creators[0].lastName == "Editors") {
```

Comment:
just to make sure -- this would always be the first author? No need to cycle through creatores?

Reply:
yep this is just so the generic "MCV Editors" byline doesn't get digested into "Editors, MCV"

don't believe I've seen two+ authors on this site, but the generic byline wouldn't be used with a second author, fwiw